packages/manager/modules/vrack/src/add/vrack-add.html

```diff
@@ -0,0 +1,66 @@
+<div id="vrack-add" class="row">
+
+    <!-- init loading... -->
+    <div data-ng-if="VrackAddCtrl.loaders.loading" class="col-xs-10 col-xs-offset-1 col-md-11 page-margins">
+        <div class="text-center">
+            <oui-spinner></oui-spinner>
+        </div>
+    </div>
+
+    <div data-ng-if="!VrackAddCtrl.loaders.loading"  class="col-xs-10 col-xs-offset-1 col-md-11 page-margins">
+        <div class="page-header">
+            <h1 data-translate="vrack_title"></h1>
+        </div>
+
+        <h5 class="top-space-m20" data-translate="vrack_subtitle" data-ng-if="!VrackAddCtrl.loaders.validationPending"></h5>
+
+        <div class="row top-space-m20">
+            <div class="col-xs-10 col-xs-offset-1"
+                 data-ng-if="!VrackAddCtrl.loaders.loading && !VrackAddCtrl.model.error">
+
+                <!-- validationPending -->
+                <div class="top-space-m20" data-ng-if="VrackAddCtrl.loaders.validationPending">
+                    <h2 class="text-center" data-translate="vrack_validationPending" data-translate-values="{url: VrackAddCtrl.model.purchaseOrderUrl}"></h2>
+                    <p class="text-center fs12" data-translate="vrack_order_delay"></p>
+                </div>
+
+                <!-- form -->
+                <div data-ng-if="!VrackAddCtrl.loaders.validationPending">
+                    <!-- contracts -->
+                    <div class="row" data-ng-if="!VrackAddCtrl.model.contractsAccepted && VrackAddCtrl.model.agreements.length"
+                         data-contracts="VrackAddCtrl.model.agreements"
```

Comment:
By using `contracts` directive it mean that you are linked to `cloud/client/components/contracts/contracts.directive.js`.

I'm not sure that we are providing a standalone module. Am I right?